apps/aclcli/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 2.8)
+project(unabto-tunnel C ASM)
+
+
+set(UNABTO_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+
+option(LOG_ALL "Print all unabto log statements" OFF)
+
+include(${UNABTO_ROOT}/build/cmake/unabto_project.cmake)
+
+INCLUDE (CheckIncludeFiles)
+# usage: CHECK_INCLUDE_FILES (<header> <RESULT_VARIABLE> )
+
+
+if(EXTERNAL_PREFIX) 
+  include_directories(${EXTERNAL_PREFIX}/include)
+  link_directories(${EXTERNAL_PREFIX}/lib)
+endif()
+
+
+include_directories(
+  # make it possible to override unabto_config.h with a supplied file
+  # to the build tree.
+  ${CMAKE_CURRENT_BINARY_DIR}
+
+  ${unabto_include_directories}
+  //${UNABTO_ROOT}/src/modules/fingerprint_acl
```

Comment:
// is not a valid comment style this just gives extra slashes in start of the filename